rdr_service/services/jira_utils.py

```diff
@@ -34,15 +37,34 @@ def __init__(self):
 
         self._connect_to_jira()
 
+    def set_jira_credentials_from_config(self):
+        """
+        Sets the Jira API username and password
+        based on whether set in environment or in config
+        :return:
+        """
```

Comment:
I thought we talked about just putting the JIRA credentials in the App Configs instead of reading a separate file.

Reply:
Yes, we did. I was running into issues with getting the credentials during testing without putting them in the .current_config.json or config_dev.json. 
This would have exposed them to the repo since those files are still public.

Reply:
The trick to that is to read them from the local environment variables before reading from the config.  So you would check `os.environ` to see if the JIRA credentials are there and if so use them for local testing.

Reply:
I've implemented the update to use the app configs.